src/V3Width.cpp

```diff
@@ -5180,6 +5199,39 @@ class WidthVisitor final : public VNVisitor {
             // if (debug()) nodep->dumpTree("-    assign: ");
             AstNodeDType* const lhsDTypep
                 = nodep->lhsp()->dtypep();  // Note we use rhsp for context determined
```

Comment:
I noticed no skipDref in the new widthUnpacked function, and none here.  Might be ok, but please have part of the test use a typedef for the array type to make sure it works (unless is already and I missed it), thanks.